<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:Cornsilk;</string>
  </property>
  <widget class="QLineEdit" name="rezult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 4px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyPlusMinus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>+/-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyComa">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key0">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyRezult">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyPlus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>190</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyMinus">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyDel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyUm">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>101</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyBack">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="keyUm_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:50%;
background-color: LightSkyBlue;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>keyUm_3</sender>
   <signal>clicked()</signal>
   <receiver>rezult</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
